src/python/foglamp/configuration_manager.py

```diff
@@ -0,0 +1,330 @@
+# -*- coding: utf-8 -*-
+"""FOGLAMP_PRELUDE_BEGIN
+{{FOGLAMP_LICENSE_DESCRIPTION}}
+
+See: http://foglamp.readthedocs.io/
+
+Copyright (c) 2017 OSIsoft, LLC
+License: Apache 2.0
+
+FOGLAMP_PRELUDE_END
+"""
+
+import logging
+import psycopg2
+import aiopg.sa
+import sqlalchemy as sa
+import asyncio
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.sql import text
+
+"""Configuration Manager
+"""
+
+__author__ = 'Ashwin Gopalakrishnan'
+__version__ = '${VERSION}'
+
+_configuration_tbl = sa.Table(
+    'configuration',
+    sa.MetaData(),
+    sa.Column('key', sa.types.CHAR(10)),
+    sa.Column('description', sa.types.VARCHAR(255)),
+    sa.Column('value', JSONB),
+    sa.Column('ts', sa.types.TIMESTAMP)
+)
+"""Defines the table that data will be inserted into"""
+
+_connection_string = 'postgresql://foglamp:foglamp@localhost:5432/foglamp'
+_logger_name = 'configuration-manager'
```

Comment:
logger = logging.getLogger(__name__) and use `logger.exception(.....` as needed.